apps/iis.go

```diff
@@ -43,10 +43,28 @@ func (r MetricsReceiverIis) Pipelines() []otel.Pipeline {
 				},
 			},
 			Processors: []otel.Component{
-				otel.NormalizeSums(),
+				otel.TransformationMetrics(
+					otel.FlattenResourceAttribute("iis.site", "site"),
+					otel.FlattenResourceAttribute("iis.application_pool", "app_pool"),
+				),
+				// Drop all resource keys; Must be done in a separate transform,
```

Comment:
`RetainResource` drops all resource keys except the specified keys; Since no keys are specified here, all keys are dropped.

If you have a better name for the function, I'm happy to change it, because maybe it's confusing.

Reply:
Oh, I see now, thanks. I just parsed the comment wrong I think. LGTM